src/Products/Signature/Controllers/SignatureApiController.cs

```diff
@@ -35,34 +32,9 @@ namespace GroupDocs.Signature.MVC.Products.Signature.Controllers
     [EnableCors(origins: "*", headers: "*", methods: "*")]
     public class SignatureApiController : ApiController
     {
-        private static Common.Config.GlobalConfiguration GlobalConfiguration;
-        private List<string> SupportedImageFormats = new List<string>() { ".bmp", ".jpeg", ".jpg", ".tiff", ".tif", ".png" };
-        private static SignatureHandler SignatureHandler;
-        private DirectoryUtils DirectoryUtils;
-
-        /// <summary>
-        /// Constructor
-        /// </summary>
-        public SignatureApiController()
-        {
-            // get global configurations 
-            GlobalConfiguration = new Common.Config.GlobalConfiguration();
-            // initiate DirectoryUtils
-            DirectoryUtils = new DirectoryUtils(GlobalConfiguration.Signature);
-            // create signature application configuration
-            SignatureConfig config = new SignatureConfig();
-            config.StoragePath = DirectoryUtils.FilesDirectory.GetPath();
-            config.CertificatesPath = DirectoryUtils.DataDirectory.CertificateDirectory.Path;
-            config.ImagesPath = DirectoryUtils.DataDirectory.ImageDirectory.Path;
-            config.OutputPath = DirectoryUtils.GetTempFolder().GetPath();
-            // initialize instance for the Image mode
-            SignatureHandler = new SignatureHandler(config);
-            License license = new License();
-            if (File.Exists(GlobalConfiguration.Application.LicensePath))
-            {
-                license.SetLicense(GlobalConfiguration.Application.LicensePath);
-            }
-        }
+        private static readonly Common.Config.GlobalConfiguration GlobalConfiguration = new Common.Config.GlobalConfiguration();
+        private readonly List<string> SupportedImageFormats = new List<string>() { ".bmp", ".jpeg", ".jpg", ".tiff", ".tif", ".png" };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/gh/groupdocs-signature/GroupDocs.Signature-for-.NET-MVC/pullRequest?prid=5325146)